src/main/java/frc/trigon/robot/commands/factories/AmpCommands.java

```diff
@@ -0,0 +1,73 @@
+package frc.trigon.robot.commands.factories;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.trigon.robot.RobotContainer;
+import frc.trigon.robot.commands.CommandConstants;
+import frc.trigon.robot.constants.AutonomousConstants;
+import frc.trigon.robot.constants.FieldConstants;
+import frc.trigon.robot.subsystems.ampaligner.AmpAlignerCommands;
+import frc.trigon.robot.subsystems.ampaligner.AmpAlignerConstants;
+import frc.trigon.robot.subsystems.intake.IntakeCommands;
+import frc.trigon.robot.subsystems.intake.IntakeConstants;
+import frc.trigon.robot.subsystems.pitcher.PitcherCommands;
+import frc.trigon.robot.subsystems.pitcher.PitcherConstants;
+import frc.trigon.robot.subsystems.shooter.ShooterCommands;
+import frc.trigon.robot.subsystems.shooter.ShooterConstants;
+import frc.trigon.robot.subsystems.swerve.SwerveCommands;
+
+public class AmpCommands {
+    public static Command getAutonomousScoreInAmpCommand() {
+        return new ParallelCommandGroup(
+                getAutonomousPrepareForAmpCommand(),
+                GeneralCommands.runWhenContinueTriggerPressed(getFeedToAmpCommand()),
+                getPathfindToAmpCommand().andThen(GeneralCommands.duplicate(CommandConstants.FACE_AMP_COMMAND))
+        );
+    }
+
+    public static Command getScoreInAmpCommand() {
+        return new ParallelCommandGroup(
+                getPrepareForAmpCommand(),
+                GeneralCommands.runWhenContinueTriggerPressed(getFeedToAmpCommand()),
+                GeneralCommands.duplicate(CommandConstants.FACE_AMP_COMMAND)
+        );
+    }
+
+    /**
+     * Creates a command that prepares to score in the amp for the autonomous score in amp command.
+     * This command only activates if you are within a certain distance from the amp to ensure that the robot doesn't hit the stage.
```

Comment:
`This command waits until we're within a certain distance from the amp to ensure that the robot doesn't hit the stage. 
We're only waiting for the autnomous placement to allow for driver judgment in the normal placement, and so it won't interfere in a case where pose estimation isn't working.